perfkitbenchmarker/linux_benchmarks/fio_benchmark.py

```diff
@@ -394,7 +396,9 @@ def GetLogFlags():
                    (FLAGS.fio_iops_log, '--write_iops_log=%(filename)s',),
                    (collect_logs, '--log_avg_msec=%(interval)d',)]
   fio_command_flags = ' '.join([flag for given, flag in fio_log_flags if given])
-  return fio_command_flags % {'filename': PKB_FIO_LOG_FILE_NAME,
+  now = time.time()
+  filename_base = '%s_%s' % (PKB_FIO_LOG_FILE_NAME, str(now))
```

Comment:
I haven't had problems with having a dot in the filename, and it could potentially cause problems if the sampling interval is small enough. Do you feel strongly about this?